ui/src/pages/Home/ProjectsView/hooks/useGetProjects.tsx

```diff
@@ -0,0 +1,37 @@
+import {useEffect, useState} from "react";
+import {IProjectDetails} from "../../../../interfaces";
```

Comment:
I'd take a look at aliases for the FE:

https://github.com/gsoft-inc/craco
https://www.npmjs.com/package/craco-alias